sonar-php-plugin/src/main/java/org/sonar/plugins/php/reports/ExternalReportFileHandler.java

```diff
@@ -97,4 +99,25 @@ private boolean isKnownFile(String path) {
     return fileSystem.hasFiles(fileSystem.predicates().hasPath(path));
   }
 
+  @CheckForNull
+  private static String separatorsToSystem(@Nullable String path) {
+    if (path == null) {
+      return null;
+    } else {
+      return isSystemWindows() ? separatorsToWindows(path) : separatorsToUnix(path);
+    }
+  }
+
+  private static boolean isSystemWindows() {
+    return File.separatorChar == '\\';
+  }
+
+  private static String separatorsToUnix(@Nullable String path) {
+    return path != null && path.indexOf(92) != -1 ? path.replace('\\', '/') : path;
+  }
+
+  private static String separatorsToWindows(@Nullable String path) {
+    return path != null && path.indexOf(47) != -1 ? path.replace('/', '\\') : path;
+  }
```

Comment:
This is copied from `org.apache.commons.io.FileUtils`. I won't be able to create coverage without changing the behavior of `File.separatorChar` for this purpose.